Include/internal/pycore_lock.h

```diff
@@ -28,6 +28,9 @@ extern "C" {
 // Typical initialization:
 //   PyMutex m = (PyMutex){0};
 //
+// Or initialize as global variables:
+//   PyMutex m;
```

Comment:
Let's tweak the example so it's clear that we're not declaring a local variable.

```suggestion
// Or initialize as global variables:
//   static PyMutex m;
```